.github/workflows/gradle-wrapper.yaml

```diff
@@ -0,0 +1,15 @@
+name: gradle-wrapper
+
+on:
+  pull_request:
+    paths:
+      - 'gradlew'
+      - 'gradlew.bat'
+      - 'gradle/wrapper/'
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: gradle/wrapper-validation-action@v1
```

Comment:
Need to think about this more. Pretty sure I want this back in the `build` step so that it catches pushes too. Also that way it block publishing if validation fails.

Reply:
This action is only useful when Gradle wrapper is changed, we don't need to run it for every check.

Reply:
Nah, revert. I want the proof authenticity to be in the same build that publishes stuff.

Reply:
cf4c91736565269d927507343ac1ebcfac460caf